### DISCUSSION TITLE
Display stops updating after 24-48h

### DISCUSSION THREAD

**Discussion by mcpzi** [pixelit-project/PixelIt#70] [Q&A] [answered]
Hello,

I'm using a d1 mini with the 0.3.7 firmware (with the luminosity sensor). It's working well overall but I have two issues :

1. After being powered on for 24-48h the display stops updating and I cannot access the web interface anymore, I need to unplug and replug the unit to get it working again
2. The WiFi credentials are often lost after reboot, I need to connect to the PixelIT hotspot to enter the credentials

Thanks for working on this project !

**Comment by o0shojo0o** [comment]
Hi, 

I would for both problems basically first zero the flash (with the CleanFlash.bin is included in the download of the Flasher).
https://docs.bastelbunker.de/pixelit/firmware.html#flashen

And then go to the latest version.
https://github.com/o0shojo0o/PixelIt/releases

Maybe your problems are already solved.

**Reply by mcpzi** [reply]
Thanks for the quick reply, I've first flashed with CleanFlash.bin and then updated to the latest version (0.3.9) but both problems are not solved.

However the display crashes less often but from time to time the display lags a lot (the colon between hours and minutes takes almost 10 seconds to blink). During that time the web interface is still accessible but doesn't show any values (they're stuck on "loading"). Then after 10-20 minutes it gets back to normal.

**Comment by o0shojo0o** [comment, marked as answer]
Hmmm... that's very strange, I haven't heard such a behavior from any other user until now. 
I have no idea what the reason could be.
Could you maybe test another d1 mini?

**Reply by o0shojo0o** [reply]
You could also try the new version, which fixes e.g. errors with the clock display. 
https://github.com/o0shojo0o/PixelIt/releases/tag/0.3.10

**Reply by mcpzi** [reply]
Thanks ! I do not have another d1 mini right now so I'll test the new firmware

**Reply by mcpzi** [reply]
Looks like it did the trick ! I haven't experienced a crashed or lag since the flash with 0.3.10

**Reply by o0shojo0o** [reply]
Yes, very nice to hear!
